t/ui/27-plugin_obs_rsync_obs_status.t

```diff
@@ -153,5 +232,35 @@ $t->get_ok('/admin/obs_rsync/queue')->status_is(200, 'jobs list')->content_like(
 $t->get_ok('/admin/obs_rsync/')->status_is(200, 'project list')->content_like(qr/published/)->content_like(qr/dirty/)
   ->content_like(qr/publishing/);
 
+subtest 'build service ssh authentication' => sub {
+    is($helper->is_status_dirty('ProjWithAuth'), 1, 're-authenticate with ssh auth');
+};
+
+subtest 'build service authentication: signature generation' => sub {
+    $mocked_time = 1664187470;
+    note "time right now: " . time();
+
+    is(time(), $mocked_time, 'Time is not frozen!');
+
+    is($helper->is_status_dirty('ProjTestingSignature'), 1, 'signature matches fixture');
+    $mocked_time = undef;
+};
+
+subtest 'build service authentication: error handling' => sub {
+    $ssh_keyfile->remove();
+    throws_ok {
+        $helper->is_status_dirty('ProjTestingSignature')
+    }
+    qr/SSH Key File not found at/, 'Key detection logic failed (not existing key file)';
+
+    $ssh_keyfile->touch();
```

Comment:
It looks like that does not work. I did look at the output of `ls -l $ssh_keyfil` after that and the file wasn't there.
Maybe a question for @kraih ?

Reply:
No clue why the file wouldn't exist after a `->touch` call.

Reply:
Should I: 
```suggestion
    path($sssh_keyfile)->touch;
```
?

Reply:
That seems to work, yeah